harness-delegate-ng/templates/secret.yaml

```diff
@@ -8,4 +8,4 @@ metadata:
 type: Opaque
 data:
   # Base 64 encoded account secret
```

Comment:
please update: chart.yaml, index.yaml and packaged chart in /charts

Reply:
@AnupamIO  I was thinking the chart yaml index yaml changes are about bumping up versions. this will happen at release time. @Vbhatia20 's pipeline will take care of this